0006-cmx2cdp-protocol/README.md

```diff
@@ -105,9 +105,9 @@ _"config-response"_ message structure:
 - `contentType` (string, required) - content type of configuration, e.g.: json, protobuf.
 - `configId` (string, required) - unique identifier of endpoint configuration.
 - `statusCode` (number, required) - status code that holds meaningful information about the result of inbound message processing.
-- `reasonPhrase` (string, optional) - status code that holds meaningful information about the result of inbound message processing.
+- `reasonPhrase` (string, required) - status code that holds meaningful information about the result of inbound message processing.
 - `content` (byte[], optional) - content with configuration data.
-
+- `isApplied` (boolean, required) - shows if client has latest configuration (true if he does).
```

Comment:
We only store the latest config id and I think it makes no sense to persist the applied config ID on the endpoint-application version level. The server will not be able to do anything with `lastAppliedConfigId`. The only possible operation is to compare `lastAppliedConfigId` with the current config ID. I suggest to modify persistence layer and config response message format if we'll need such info.